src/promptflow/tests/sdk_cli_azure_test/unittests/test_azure_cli_perf.py

```diff
@@ -56,10 +38,16 @@ def operation_scope_args(subscription_id: str, resource_group_name: str, workspa
     ]
 
 
-@pytest.mark.usefixtures("mock_get_azure_pf_client")
 @pytest.mark.perf_monitor_test
-class TestAzureCliTimeConsume:
-    def test_pfazure_run_create(self, operation_scope_args, time_limit=30):
+@pytest.mark.usefixtures(
+    "mock_get_azure_pf_client",
+    "mock_set_headers_with_user_aml_token",
+    "single_worker_thread_pool",
+    "vcr_recording",
+)
+class TestAzureCliPerf:
```

Comment:
might need to move this test to E2E test folder, without previous mock, this test will run against server, then unit test folder is not a good place to locate

Reply:
yes.